samples/BenchmarkDotNet.Samples.Runner/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "BenchmarkDotNet": "0.10.1-develop",
```

Comment:
you can make it a reference to the project, then there is no need to specify the version (which would require change with every update)

```json
"BenchmarkDotNet": {
          "target": "project"
        }
```